Utilities/bin_patch.cpp

```diff
@@ -396,7 +396,7 @@ std::size_t patch_engine::apply_patch(const std::string& name, u8* dst, u32 file
 			{
 				offset -= ls_addr;
```

Comment:
I wanted to move this line below the check, but forgot.
(would also be a slight optimization compared to your fix)

Reply:
Right, that's better

Reply:
alternatively keep yours and add mine